### DISCUSSION TITLE
Build path specifies execution environment JavaSE-15. There are no JREs installed in the workspace that are strictly compatible with this environment.

### DISCUSSION THREAD

**Discussion by RoiSoleil** [eclipse-theia/theia#11976] [Q&A]
How do i change JRE installed in Theia ?

**Comment by msujew** [comment]
Hey @RoiSoleil,

Theia doesn't come with any build tools, runtime environments or other compilers. We only provide the framework for building IDEs. Are you using the [RedHat extension](https://github.com/redhat-developer/vscode-java) for Java support in your IDE? Maybe you can ask them.
